src/components/side_panel/chart/scorecard_chart_panel/scorecard_chart_design_panel.xml

```diff
@@ -3,10 +3,16 @@
     <t t-set="color_up">Color Up</t>
     <t t-set="color_down">Color Down</t>
     <div>
-      <ChartColor
-        currentColor="props.definition.background"
-        onColorPicked="(color) => this.setColor(color, 'backgroundColor')"
-      />
```

Comment:
What about keeping `ChartColor` (that would use `PanelColorPicker`) to avoid copy pasting this part of template over and over again.

Reply:
With Anthony's upcoming refactoring of the chart design panel ([3792009](https://www.odoo.com/web#id=3792009&menu_id=4720&cids=1&action=333&active_id=2328&model=project.task&view_type=form)), the backround most likely won't be in a Section. 

So a ChartBackgroundColor component would just be a flex div + a span. At this point a component would be very overkill, and probably create more lines of code than it saves. We can have a look at it once his task is ready to reconsider if it's worth

Reply:
ok :)